example/vulkan_example.cpp

```diff
@@ -0,0 +1,48 @@
+#include "tensorflow/lite/kernels/register.h"
+#include "tensorflow/lite/delegates/vulkan/vulkan_delegate.h"
+#include "tensorflow/lite/delegates/vulkan/vulkan_delegate_adaptor.h"
+
+#include "tensorflow/lite/c/common.h"
+
+int main(int argc, char* argv[])
+{
+    printf("Loading Linear model: y = 2x-1\n");
+    std::unique_ptr<tflite::FlatBufferModel> model = tflite::FlatBufferModel::BuildFromFile("linear.tflite");
+
+    if(!model){
```

Comment:
* It's important to choose a coding standard from the very beginning. For C++ I like Google's style guide, for example. You can use `clang-format --style=Google file.cpp`.

* The other important part, is to watch naming standard.  You can find google's [here](https://google.github.io/styleguide/cppguide.html).

* I recommend running `cpplint` to ensure you are using good c++ practices.